components/style/lib.rs

```diff
@@ -23,7 +23,6 @@
 //! [cssparser]: ../cssparser/index.html
 //! [selectors]: ../selectors/index.html
 
-#![deny(warnings)]
```

Comment:
Let's leave these explicit instances of denying warnings, since developers have grown used to developing with them present.

Reply:
@jdm Shouldn’t we rather be consistent between crates? As far as I’m concerned it’s an historical accident that some crates have it and not others, not something useful.

Reply:
I don't personally care, but at least nsstring is vendored code, so I'd rather leave it in that crate at least.